File: JavaBasic/144_count_uppercase.java
// 144	144_count_uppercase.java	Count the number of uppercase letters in a given string.
public class _144_count_uppercase {
    public static void main(String[] args) {
        String str = "Hello World JS";
        int count = 0;
        for (char c : str.toCharArray()) if (Character.isUpperCase(c)) count++;
        System.out.println("Uppercase count: " + count);
    }
}

File: JSBASIC/144_count_uppercase.js
// 144	144_count_uppercase.js	Count the number of uppercase letters in a given string.

let str = "Hello World JS";
let count = 0;
for (let i = 0; i < str.length; i++) {
    if (str[i] >= 'A' && str[i] <= 'Z') count++;
}
console.log("String:", str);
console.log("Uppercase letters:", count);
